plugins/scaffolder-node/src/actions/util.ts

```diff
@@ -109,6 +110,30 @@ export const parseRepoUrl = (
   return { host, owner, repo, organization, workspace, project };
 };
 
+/**
+ * @public
+ */
+export const parseJSON = (objectString: string): JsonSpec => {
+  let parsed;
+  try {
+    parsed = JSON.parse(objectString);
+  } catch (error) {
+    throw new InputError(`Invalid object passed to publisher, ${error}`);
+  }
+
+  const results: JsonSpec = {};
+
+  if (parsed) {
+    for (const key in parsed) {
+      if (parsed.hasOwnProperty(key)) {
+        results[key] = parsed[key];
+      }
+    }
+  }
+
+  return { ...results };
```

Comment:
this shouldn't be needed, right? 🤔 

```suggestion
  return parsed;
```

Reply:
The if is not really necessary there, we were working on other implementations and I forgot to remove the if :/